src/cpu-template-helper/src/dump/aarch64.rs

```diff
@@ -1,8 +1,8 @@
 // Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-use vmm::guest_config::templates::aarch64::{RegisterModifier, RegisterValueFilter};
-use vmm::guest_config::templates::{CpuConfiguration, CustomCpuTemplate};
+use vmm::guest_config::templates::aarch64::RegisterModifier;
```

Comment:
nit: typo in commit message `serialiaztion` (twice).

Reply:
fixed